packages/amplication-client/src/VersionControl/CommitData.scss

```diff
@@ -1,5 +1,5 @@
 @import "../style/index.scss";
-$metadata-width: 150px;
+$metadata-width: 180px;
```

Comment:
The CommitData component is also used in the CommitSelector. just making sure we tested the impact in both places

Reply:
checked, it's OK there